nikolay_shapurov/task_02/graph.cpp

```diff
@@ -1,12 +1,10 @@
 #include "graph.hpp"
 
 void Graph::add_vertex() {
-  static VertexId genId = 0;
-  vertexVec_.push_back(Vertex(genId++));
+  vertices_.emplace_back(Vertex(get_new_vertex_id()));
```

Comment:
В `emplace` нужно передавать список аргументов, такой же, как в конструктор, чтобы он мог сам сконструировать объект.
```suggestion
  vertices_.emplace_back(get_new_vertex_id());
```